import-sample.md

```diff
@@ -1,6 +1,6 @@
-# Import HISPlayer visionOS Sample Beta
+# Import HISPlayer visionOS Sample
 
-Please, download the sample here -> [**HISPlayer visionOS Sample Beta**](https://downloads.hisplayer.com/Unity/AllPlatforms/HISPlayer_Sample_3.3.3_Beta.unitypackage) 
+Please, download the sample here -> [**HISPlayer visionOS Sample**](https://downloads.hisplayer.com/Unity/visionOS/HISPlayer_visionOS_Sample.unitypackage) 
```

Comment:
@StAincrad dont forget to update the [Sample page ](https://github.com/HISPlayer/UnitySamples)

Reply:
@ivder Yes, I already have a draft for the Sample page, thanks